open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -119,12 +119,17 @@
    "outputs": [],
```

Comment:
<p>You can leave some blank to line 15-19 as well. But please remember to give some inline comments if you make them blank.</p><p> </p><p>same to below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>DONE</p>